golos.publication/golos.publication.cpp

```diff
@@ -1083,6 +1087,7 @@ void publication::set_limit(
 void publication::set_rules(const funcparams& mainfunc, const funcparams& curationfunc, const funcparams& timepenalty,
     uint16_t maxtokenprop, eosio::symbol tokensymbol
 ) {
+    eosio::check(tokensymbol == token::get_supply(config::token_name, tokensymbol.code()).symbol, "symbol precision mismatch");
```

Comment:
There is no need in separate db request, symbol of `unclaimed_funds` can be used